### DISCUSSION TITLE
Paraview display scalars, such as "damage", it shows no color!  why?   what happened?

### DISCUSSION THREAD

**Discussion by IntePNum** [peridigm/peridigm#324] [General]
Paraview display scalars, such as "damage", it shows no color!  why?   what happened?
![damage_noclor](https://github.com/peridigm/peridigm/assets/108223279/1f48ebe7-f2a4-444d-8f1d-5445ad4a5a5f)

**Comment by djlittl** [comment]
I don't think "Point Gaussian" is a valid option for meshfree data.  Switch it to "Surface".  I tried "Points" and "Surface With Edges" and those worked too.

**Reply by IntePNum** [reply]
Thanks for your reply. I tried the "points" and "surface",They don't work. 
It is related to paraview version? my version is 5.11.1. 
what is your version？

![point](https://github.com/peridigm/peridigm/assets/108223279/89fea5ab-9b83-48f2-9bb7-eb68a50c34d5)

**Reply by oldninja** [reply]
I think you need to add the "cell data to point data" filter and then do that about the surface- if I recall correctly.

**Reply by IntePNum** [reply]
Thank you very very very much!  how can you be so clever!  resepct!!!!!

**Reply by IntePNum** [reply]
Hello, Prof. Littlewood, I noted the statement in the code as “Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are  met:”?  what that mean?  I don't quite understand about intellectual property. Does this restrict me from using source code to publish articles?
![state_ment](https://github.com/peridigm/peridigm/assets/108223279/7d8e3a5d-b7f4-4beb-8de4-a237cd868b16)

**Comment by djlittl** [comment]
I am also not an expert in intellectual property.  But I am 100% confident that you can use Peridigm to publish a paper.

The language above is the so-called "three-clause BSD license".  A lot of information on it can be found on the web.  The Wikipedia page might be a good place to start:  https://en.wikipedia.org/wiki/BSD_licenses

The 3-clause BSD license is not meant to be particularly restrictive.  It's meant to protect the license holder from things like other parties claiming they wrote the code, selling it for profit, or holding the authors financially liable if the simulation results are not correct.

Like any open-source code, if you use Peridigm for a paper, please acknowledge that you did so and cite it in the bibliography.  The best way to cite Peridigm is the newly-published paper in JPER or the original Users Manual.  Links to both can be found on the main Peridigm GitHub page.

Beyond that, you don't need anyone's permission to use Peridigm.  It's great if you're able to use it for a research paper.  Ideally, users will join the Peridigm community here on GitHub to contribute code, tests, etc., and help answer each others' questions.

**Reply by IntePNum** [reply]
Thank!!!!!! respect again!!!!!!!  yo
